### ISSUE TITLE
Breaking changes on intl-tel-input 16.x

### ISSUE THREAD

**Issue by asantos3** [webcat12345/ngx-intl-tel-input#211]
On the 16.0.0 [intl-tel-input](https://github.com/jackocnr/intl-tel-input/) did a few breaking changes that broke this package: 
> * Switched CSS class names to use BEM, so you shouldn't get any more clashes with generic class names like "highlight"
> * All NANP countries now have just +1 as their dialCode, when some used to include the area code e.g. Barbados used to have dial code +1246 and now has just +1.

Some issues like #209 might be open due to this.

**Comment by pasevin** [comment]
Yep, this is why we're still on ```"intl-tel-input": "^14.1.0"```.
If anyone wants to adapt this lib to v16.0.0, please submit a PR. Appreciate :)

**Comment by errier** [comment]
Please also update https://www.npmjs.com/package/ngx-intl-tel-input page.
npm install intl-tel-input --save 
=>
npm install intl-tel-input@14.1.0 --save

**Comment by pasevin** [comment]
Done :)
